appcues/src/main/res/values/strings.xml

```diff
@@ -56,6 +56,7 @@
 
     <string name="appcues_debugger_info_title">Info</string>
     <string name="appcues_debugger_info_fonts">Available Fonts</string>
+    <string name="appcues_debugger_plugins_title">Plugins</string>
```

Comment:
oh, what is Plugins vs Custom components here, I may have misunderstood in previous comment.

Reply:
Plugins is the inicial option in the front panel, then we have a list of CustomComponents  and maybe some other things in the future?

<img width="348" alt="image" src="https://github.com/user-attachments/assets/e85138a4-7103-494f-9122-5883bf9b2cfd">

Reply:
got it, ok, as long as we are consistent on ios/android, all good